example.js

```diff
@@ -5,30 +5,72 @@
 const Ws = require('ws')
 const ws = new Ws('ws://localhost:8899')
 
+const terms = require('bitfinex-terminal-terms-of-use')
```

Comment:
@AntonNazarenko the idea is this:

users have to agree to the TOS before using bitfinex terminal. so on first start the ui will show the terms of service, and when the user accepts it, store it in the settings.

when the TOS have updated, an error is returned and the user has to update the honey framework currently, for that we need a small window that informs the user in those cases.

if the user has not accepted the TOS yet, when they go to backtest and select the Terminal specific elements, the TOS window is shown again and they have to agree to use the features. this is again stored in the settings.